### PR TITLE
Switch to rubocop-airbnb

### PR DESCRIPTION
**Pull request by pariser**

### CODE REVIEW

**Comment by santiagorodriguez96** [comment]
Code looks good to me! 💯 

We could wait until #35 gets in so we can have a successful build before merging 🙂

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.3.1
+2.6.8
```
Comment (santiagorodriguez96) [inline .ruby-version:1]: It's recommended to use the oldest supported Ruby version here, so that `rubocop` can analyze    the files accordingly 👀

Comment (pariser) [reply .ruby-version:1]: Running rubocop against ruby2.5 syntax with this change https://github.com/airbnb/sparsam/pull/33/files#diff-4f894049af3375c2bd4e608f546f8d4a0eed95464efcdea850993200db9fef5cR2

To be honest, we should probably drop support for 2.5 and 2.6 soon, they're past EOL (2.5 is _well past_ EOL).

**Review by santiagorodriguez96 — APPROVED** [review]
Looks good to me! 💯

**Review by Umofomia — APPROVED** [review]
Looks good to me. You're standardizing on using the rules from `rubocop-airbnb` and addressed existing violations of the new rules, excluding cases of `Airbnb/OptArgParameters` in public interfaces that cannot be easily changed.

**Pull request merged**